### DISCUSSION TITLE
Sharing filters (Filters pannel) is not working - custom module

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#15374] [How to] [answered]
Guys, I running 6.2, created filter (in back-end filters panel), shared it with certain role (non admin) and this filter doesn't not want to show up for user. That is for custom module. Does anyone got the same? System is perfectly configured and on pretty powerful resources. Asking here before reporting, so no necessary issue will be created

**Comment by s2b-git** [comment]
It not works in custom module indeed - on GitDev shared filter for custom module showing up but returning "Entered invalid value". I created 'test' module. Please check on gitdev installation

**Reply by rskrzypczak** [reply]
I checked and I don't see any errors here.
Please add a more detailed description of how to trigger this.

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
To clarify - "Discussions" are not meant to be used for reporting errors!!! 
You can ask questions, discuss whether something is an error or not, how things work, but errors MUST BE reported in "issues", according to the proper category and template that contains all necessary information that we need. 
Templates make it easier to diagnose the errors/problems and we keep updating based on the reported errors.  
#13908
